niceman/distributions/vcs.py

```diff
@@ -79,22 +79,24 @@ class VCSRepo(SpecObject):
     path = attrib(default=attr.NOTHING)
     files = attrib(default=attr.Factory(list))
 
-    @property
-    def identifier(self):
-        try:
-            return getattr(self, self._identifier_attribute)
-        except AttributeError:
-            # raised if _identifier_attribute is not defined, but this means
-            # (to the caller) that identifier is not defined
-            msg = "%s instance has no attribute 'identifier'" % self.__class__
-            raise AttributeError(msg)
-
+    _diff_fields = ('path',)
+
+    # @property
+    # def identifier(self):
+    #     try:
+    #         return getattr(self, self._identifier_attribute)
+    #     except AttributeError:
+    #         # raised if _identifier_attribute is not defined, but this means
+    #         # (to the caller) that identifier is not defined
+    #         msg = "%s instance has no attribute 'identifier'" % self.__class__
+    #         raise AttributeError(msg)
+    #
```

Comment:
IMO there's no reason to leave commented out code unless it comes with a TODO comment.

Reply:
A comment later in the code referred to the block. :)

Gone.